### PR TITLE
Staging sites: Add feature to plans

### PR DESCRIPTION
**Pull request by kozer**
Related to https://github.com/Automattic/dotcom-forge/issues/1741

## Proposed Changes

This diff adds the appropriate changes for the new staging site feature as described [here](https://docs.google.com/spreadsheets/d/1cPh9HIVfTwUR--H05ncU4LnTuIKVD8kggg29MBDzu6U/edit#gid=0) to the plan pages

## Testing Instructions

- Visit `/plans/<your site>`
- Make sure that you see the following changes:

| **Before** | **After** |
| :---:         |    :---:       |
|plans_li_before1 | plans_li_after1|
|plans_li_before2|plans_li_after2|

UPDATE:
The wording on staging site tooltip has changed from `plural` to `singular`.

*

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by wojtekn** [comment]
@kozer code looks good, and the change looks as described in the copy spreadsheet. However, see two things: 

- in the Commerce column we say "Everything in Business, plus:", so it's not really logical to repeat "Free staging sites" there, as this is included in the Business plan already
- it seems weird when we say "Free staging sites" when we don't say that for all other features.

**Comment by kozer** [comment]
> @kozer code looks good, and the change looks as described in the copy spreadsheet. However, see two things:
> 
> * in the Commerce column we say "Everything in Business, plus:", so it's not really logical to repeat "Free staging sites" there, as this is included in the Business plan already
> * it seems weird when we say "Free staging sites" when we don't say that for all other features.

Totally agree @wojtekn . But those have already been discussed in a thread in our team's channel (for some reason pasting the link doesn't display it as it should), and this is what it's asked.

**Review by sejas — COMMENTED** [review]
I agree with @wojtekn , I prefer to remove `Free staging sites` from the eCommerce plan.

For context: p1683110195464559-slack-C04GESRBWKW

**Review by wojtekn — APPROVED** [review]

File: client/lib/plans/features-list.tsx [hunk 1]
```diff
@@ -1881,6 +1882,11 @@ export const FEATURES_LIST: FeatureList = {
 		getDescription: () =>
 			i18n.translate( 'Use familiar developer tools to manage and deploy your site.' ),
 	},
+	[ FEATURE_SITE_STAGING_SITES ]: {
+		getSlug: () => FEATURE_SITE_STAGING_SITES,
+		getTitle: () => i18n.translate( 'Free staging sites' ),
```
Comment (danielbachhuber) [inline client/lib/plans/features-list.tsx]:
```suggestion
		getTitle: () => i18n.translate( 'Free staging site' ),
```

File: client/lib/plans/features-list.tsx [hunk 2]
```diff
@@ -1881,6 +1882,11 @@ export const FEATURES_LIST: FeatureList = {
...
+		getDescription: () => i18n.translate( 'Test product and design changes in staging sites.' ),
```
Comment (danielbachhuber) [inline client/lib/plans/features-list.tsx]:
```suggestion
		getDescription: () => i18n.translate( 'Test product and design changes in a staging site.' ),
```

Comment (kozer) [reply client/lib/plans/features-list.tsx]: @danielbachhuber , in the spreadsheet, the wording "Free staging sites" is what's mentioned. Do we want that change?
Also same goes with the wpcom pr as well

Comment (danielbachhuber) [reply client/lib/plans/features-list.tsx]: @kozer Yes, each plan only gets one free staging site.

For example, it's "Free domain for one year", not "Free domains for one year".

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7989693

Thank you @kozer for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7989693).

**Pull request merged**